FINAL_COUNTDOWN.md

```diff
@@ -0,0 +1,74 @@
+This is the burn down issue to ship Glimmer 2. While there are still in flight PRs, this is known state of the world at SHA b418b95ea3d31e3a3656906e6ad784422f648c31.
+
+![Awesome 80s hair band](http://vignette2.wikia.nocookie.net/degrassi/images/d/d4/Final-Countdown.gif)
+
+# Macro Tasks
+- [ ] Complete all test migration described in #13127
+- [ ] Migrate/coalesce fragment tests #13152
+
+# Features
+
+### Infra
+- [ ] Integrate into run-loop
+
+### Helpers
+- [ ] Implement `{{partial}}`
+- [ ] Implement `{{mut}}`
+- [ ] Implement `{{textarea}}`: [Failing test 1](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L15) [Failing test 2](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L23) [Failing test 3](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/helpers/text-area-test.js#L36)
+- [ ] Implement `{{each-in}}` [Failing tests](packages/ember-glimmer/tests/integration/syntax/each-in-test.js#L23)
+
+### Currly Components
+- [ ] Implement remaining life-cycle hooks
+- [ ] Implement custom Ids for curly component invocations: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L26)
+- [ ] Implement triple curiles: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L849)
+- [ ] Implement `htmlSafe`: [Failing test](https://github.com/emberjs/ember.js/blob/b418b95ea3d31e3a3656906e6ad784422f648c31/packages/ember-glimmer/tests/integration/components/curly-components-test.js#L849)
```

Comment:
This test is not great. I will update the test to be more robust to account for differences between HTMLBars and Glimmer2. It will still fail, but at least the baseline will be set to compare them evenly.